packages/styles/framework/src/utilities/mixins/_mixins.scss

```
@@ -251,44 +290,30 @@
     }
 
     &.is-horizontal {
-      display: flex;
-      justify-content: space-between;
+      flex-direction: row;
       align-items: center;
```

Stylelint (17.3.0)

[error] Unexpected duplicate "align-items" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)